src/imitation/util/util.py

```diff
@@ -41,6 +43,7 @@ def make_vec_env(
     log_dir: Optional[str] = None,
     max_episode_steps: Optional[int] = None,
     post_wrappers: Optional[Sequence[Callable[[gym.Env, int], gym.Env]]] = None,
+    env_make_kwargs: Optional[Mapping[str, Any]] = {},
```

Comment:
As written this isn't actually `Optional` -- the code will choke if you pass in a `None`.

However, it's also bad practice to use `{}` as a default argument, since it's actually a mutable object so the default can be *changed* if you were to mutate `env_make_kwargs` anywhere.

I'd suggest changing `{}` to `None`, and then adding inside the function:
```python
env_make_kwargs = env_make_kwargs or {}
```

Reply:
I think I would prefer it have a default argument, don't want every caller to have to pass in `None` explicitly. It won't always be called from Sacred scripts.